src/SFML/Window/DRM/DRMContext.cpp

```diff
@@ -130,6 +128,308 @@ namespace
         initialized = false;
     }
 
+    void drmFbDestroyCallback(gbm_bo* bo, void* data)
+    {
+        int drmFd = gbm_device_get_fd(gbm_bo_get_device(bo));
+        sf::priv::DrmFb* fb = reinterpret_cast<sf::priv::DrmFb*>(data);
+
+        if (fb->fbId)
+            drmModeRmFB(drmFd, fb->fbId);
+
+        std::free(fb);
+    }
+
+    sf::priv::DrmFb* drmFbGetFromBo(gbm_bo& bo)
+    {
+        int drmFd = gbm_device_get_fd(gbm_bo_get_device(&bo));
+        sf::priv::DrmFb* fb = reinterpret_cast<sf::priv::DrmFb*>(gbm_bo_get_user_data(&bo));
+        if (fb)
+            return fb;
+
+        fb = reinterpret_cast<sf::priv::DrmFb*>(std::calloc(1, sizeof(*fb)));
+        fb->bo = &bo;
+
+        const sf::Uint32 width = gbm_bo_get_width(&bo);
+        const sf::Uint32 height = gbm_bo_get_height(&bo);
+        const sf::Uint32 format = gbm_bo_get_format(&bo);
+
+        sf::Uint32 strides[4] = {0};
+        sf::Uint32 handles[4] = {0};
+        sf::Uint32 offsets[4] = {0};
+        uint64_t modifiers[4] = {0};
```

Comment:
I tried to use `sf::Uint64` here instead but it failed to compile because `Uint64` didn't match the definition of `uint64_t`. I believe one of them was an `unsigned long` and the other an `unsigned long long`.

Reply:
I'd honestly rather keep it this way than learn the details of how `long long` behaves across C++ standards. Plus it's nice to avoid adding casts.